apiserver/logsink_test.go

```diff
@@ -189,8 +189,8 @@ func (s *logsinkSuite) TestLogging(c *gc.C) {
 	logPath := filepath.Join(s.LogDir, "logsink.log")
 	logContents, err := ioutil.ReadFile(logPath)
 	c.Assert(err, jc.ErrorIsNil)
-	line0 := modelUUID + " machine-0: 2015-06-01 23:02:01 INFO some.where foo.go:42 all is well\n"
-	line1 := modelUUID + " machine-0: 2015-06-01 23:02:02 ERROR else.where bar.go:99 oh noes\n"
+	line0 := modelUUID + ": machine-0 2015-06-01 23:02:01 INFO some.where foo.go:42 all is well\n"
```

Comment:
Because the entity stopped being part of the prefix and started being part of the log record. I got the impression from Menno that the log is for backup but anything that wants the data structured is reading from the DB, so a small change to the format here wouldn't be a problem. Do you think that's right?